paddle/fluid/inference/tests/api/analyzer_mobilenet_tester.cc

```diff
@@ -0,0 +1,82 @@
+/* Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
`analyzer_mobilenet_tester.cc` is similar to `analyzer_resnet50_tester.cc`. You can use `analyzer_resnet50_tester.cc` instead. You can only change in the CMakeLists.txt to use the mobilenet model.
https://github.com/PaddlePaddle/Paddle/blob/e8642c3c1fe16dde0cbd08c7c0179f6028497431/paddle/fluid/inference/tests/api/CMakeLists.txt#L81-L83
like
```
inference_analysis_api_test_with_fake_data(test_analyzer_mobilenet  
   "${INFERENCE_DEMO_INSTALL_DIR}/mobilenet" analyzer_resnet50_tester.cc "mobilenet_model.tar.gz") 
```

Reply:
I found that you no longer check if the number of passes in the resnet50 test is equal to expected like `CHECK_EQ(fc_fuse_pass_count, 1)`. It only outputs the number of passes in total. Is this okay?

Reply:
It is okay since `test_analyzer_resnet50` will be used for many different image models, such as mobilenet, or googlenet in #14346 , it could not have `CHECK_EQ(fc_fuse_pass_count, 1)`.

Reply:
Ok, I understand.